src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -121,6 +125,21 @@ public function load(array $configs, ContainerBuilder $container)
             ->addTag('security.voter');
     }
 
+    public function process(ContainerBuilder $container)
```

Comment:
I would rather put this compiler pass in a dedicated class.

We have lots of different passes in each of our bundles, and this logic is not more related to the DI extension than other passes (rather less)